<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderCreatorClass</class>
 <widget class="QWidget" name="FolderCreatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QToolButton" name="button_OK">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
     <property name="icon">
      <iconset resource="LxMemo.qrc">
       <normaloff>:/LxMemo/icons/ok.png</normaloff>:/LxMemo/icons/ok.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>input folder name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="button_cancel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
     <property name="icon">
      <iconset resource="LxMemo.qrc">
       <normaloff>:/LxMemo/icons/close.png</normaloff>:/LxMemo/icons/close.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="LxMemo.qrc"/>
 </resources>
 <connections/>
</ui>
